posts/2021-06-11-kubernetes-operator-with-java-and-quarkus.adoc

```diff
@@ -0,0 +1,712 @@
+= Kubernetes Operator with Java and Quarkus
+
+== The goal of this blog post
+A Kubernetes operator is an application that runs
```

Comment:
Looks like this is missing the rest of the sentence?

Reply:
added the missing part.